.circleci/config.yml

```diff
@@ -687,6 +687,33 @@ jobs:
           name: Release packages
           command: ./scripts/build/release-packages.sh "${CIRCLE_TAG}"
 
+  scan-docker-master:
+    docker:
+      - image: circleci/buildpack-deps:stretch
+    steps:
+      - setup_remote_docker
+      - restore_cache:
+          key: vulnerability-db
+      - run:
+          name: Install trivy
+          command: |
+            VERSION=$(
+                curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | \
+                grep '"tag_name":' | \
+                sed -E 's/.*"v([^"]+)".*/\1/'
+            )
+
+            wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
+            tar zxvf trivy_${VERSION}_Linux-64bit.tar.gz
+            sudo mv trivy /usr/local/bin
+      - run:
+          name: Scan the latest grafana master image with trivy
+          command: trivy --exit-code 1 --quiet --auto-refresh --clear-cache grafana/grafana:master
+      - save_cache:
```

Comment:
Can you overwrite the cache in CircleCI? Or will this end up using the same base cache forever?

Reply:
Yes, that's what `--clear-cache` argument is for. From https://github.com/aquasecurity/trivy#clear-image-caches: 
> The --clear-cache option removes image caches. This option is useful if the image which has the same tag is updated (such as when using latest tag).

Regarding `save_cache` it caches the vulnerability db so it doesn't have to re-generate all of it on each run.

Reply:
I was mostly thinking of the CircleCI cache. I'm wondering if the cache will ever update after it has been saved the first time? In effect, can you write the result of running trivvy to update the vuln cache. If not, the cache in circleci will get really out of date eventually.

Reply:
The CircleCI seems to be immutable, but it also is supposed to clear after 30 days. So doing this should be fine.

Reply:
You mean writing cache doesn't happen since we're failing the build?

Reply:
circleci treats cache keys as immutable, so once a given key is written you can't update it, though you can store sub-keys and those will be returned.  There was some discussion about this in #18603 and I did actually throw together a basic Dockerfile to build a trivy image with an up-to-date vuln db built into it, though the problem is that it's 2.75GB

Reply:
Me and @xlson jsut discussed this and came to the conclusion that this should be fine since seems like circleci purges cache-keys after 30 days. Will merge and then we can evaluate how it works.